docs/node/daemon/kwild.mdx

```diff
@@ -97,10 +103,14 @@ KWILD_CHAIN_RPC_LISTEN_ADDRESS="tcp://127.0.0.1:26656"
 
 kwild --root-dir "path/to/root/dir" --log.level "info" --chain.rpc.listen-addr "tcp://192.168.1.20:30000"
 ```
+
 If both flags and env variables are set, flags take precedence over environment variables. Therefore, in this example, the Kwil application uses `info` log level and RPC listen address as `tcp://192.168.1.20:30000`.
 
-Refer to the [translation guide](./config/settings#config-override) for more information on mapping the config parameters to the command line flags and environment variables.
+:::tip
+Option names in the TOML configuration file and environment variables use underscores ("_") to separate words within the name, while the command line flags use dashes ("-").
+:::
```

Comment:
![image](https://github.com/kwilteam/docs/assets/140431406/fc4cbd36-3974-413b-8de0-82742e5962bb)

Adds to the confusion a little bit, but we already had the translation with environment variables.  It just gets confusing when the docs refer to a setting generically (without saying if it's the toml or flag setting).